src/components/Questions.js

```diff
@@ -63,7 +69,29 @@ class Questions extends Component {
                                         <span>Updated: <span className='grey'>{moment(new Date(form.updated_on), "YYYYMMDD").fromNow()}</span></span>
                                     </div>
                                 </div>
-                            </Card.Content>                            
+                            </Card.Content>
+                            {
+                                type === 'admin' ?
+                                <Card.Content extra>
+                                    <Modal
+                                        trigger={<Button icon basic color='grey' labelPosition='right' onClick={this.close}>
+                                                    <Icon name='share' />
+                                                    SHARE
+                                                </Button>}
+                                        open={this.state.open}
+                                        closeOnDimmerClick={false}
+                                        closeOnEscape={false}
+                                        onClose={this.close}
+                                        closeIcon
+                                    >
+                                        <Modal.Header>Copy Link</Modal.Header>
+                                        <Modal.Content>
+                                            <Form.Input  value={'localhost:3000' + this.props.location.pathname} />
```

Comment:
@bismitaguha I suggest you add a comment above this statement as a mark so that when we deploy the website, we will remember to make this change.